tests/db/manager_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018 Patrick Totzke
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+"""Test suite for alot.db.manager module."""
+
+import tempfile
+import textwrap
+import os
+
+from alot.db.manager import DBManager
+from alot.settings.const import settings
+from notmuch import Database
+
+from .. import utilities
+
+
+class TestDBManager(utilities.TestCaseClassCleanup):
+
+    @classmethod
+    def setUpClass(cls):
+
+        # create temporary notmuch config
+        with tempfile.NamedTemporaryFile(mode='w+', delete=False) as f:
+            f.write(textwrap.dedent("""\
+                [maildir]
+                synchronize_flags = true
+                """))
+            cls.notmuch_config_path = f.name
+        cls.addClassCleanup(os.unlink, f.name)
+
+        # define an empty notmuch database in a temporary directory
+        cls.dbpath = tempfile.mkdtemp()
```

Comment:
should probably add `cls.addClassCleanup(shutil.rmtree, cls.dbpath)` here.

Reply:
Although, actually, we probably need to add `cls.addClassCleanup(cls.manager.kill_search_processes)` first, right?

Reply:
it can't hurt, yes